apps/backend/src/transaction/entities/transaction.entity.ts

```diff
@@ -1,12 +1,12 @@
 import { OneToMany, Entity, Column, PrimaryColumn } from 'typeorm';
 import { PaymentEntity } from './payment.entity';
-import { Transaction } from '../transaction.interface';
+import { Transaction } from '../interface/transaction.interface';
 import { ColumnNumericTransformer } from '../../common/transformers/numericColumnTransformer';
 
 @Entity('transaction')
 export class TransactionEntity {
   @PrimaryColumn({ unique: true })
-  id: string;
+  transaction_id: string;
```

Comment:
Updated to match the json naming convention (in order to avoid any confusion when comparing with the json for testing etc)